pom.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="UTF-8"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>stathissideris</groupId>
+  <artifactId>ditaa</artifactId>
+  <packaging>jar</packaging>
+  <version>0.11.0</version>
+  <name>ditaa</name>
+  <description>command-line utility that can convert diagrams drawn using ascii art into proper bitmap graphics</description>
+  <url>https://github.com/stathissideris/ditaa</url>
+  <licenses>
+    <license>
+      <name>GNU Lesser General Public License v3.0</name>
+      <url>https://www.gnu.org/licenses/lgpl-3.0.en.html</url>
+    </license>
+  </licenses>
+  <scm>
+    <url>https://github.com/dakusui/ditaa</url>
+    <connection>scm:git:git://github.com/dakusui/ditaa.git</connection>
+    <developerConnection>scm:git:ssh://git@github.com/dakusui/ditaa.git</developerConnection>
+    <tag>95a481e5a01d07317edc4af4cbdfb4c821090437</tag>
```

Comment:
this tag value came from some example where maven project was release.

Should not be here, as fare as I know

Reply:
Thank you for pointing out.
I'm closing this pull request since this is not necessary anymore.